.env.example

```
@@ -0,0 +1,7 @@
+# Supabase
+# https://supabase.com/dashboard/project/_/settings/api から取得
+PUBLIC_SUPABASE_URL=https://xxxxxxxxxxxxxxxxxxxx.supabase.co
+PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_SUPABASE_ANON_KEY key should go before the PUBLIC_SUPABASE_URL key

(UnorderedKey)